examples/seismic/acoustic/operators.py

```diff
@@ -1,10 +1,44 @@
+from sympy import sign
+
 from devito import Eq, Operator, Function, TimeFunction, Inc, solve
+from devito.symbolics import retrieve_functions, INT
 from examples.seismic import PointSource, Receiver
 
 
-def laplacian(field, m, s, kernel):
+def freesurface(model, eq):
+    """
+    Generate the stencil that mirrors the field as a free surface modeling for
+    the acoustic wave equation.
+
+    Parameters
+    ----------
+    model : Model
+        Physical model.
+    eq : Eq
+        Time-stepping stencil (time update) to mirror at the freesurface.
     """
-    Spacial discretization for the isotropic acoustic wave equation. For a 4th
+    lhs = eq.lhs
+    rhs = eq.rhs.evaluate
+    # Get vertical dimension and corresponding subdimension
+    z = model.grid.dimensions[-1]
```

Comment:
No need for this - can just to pulled from `zfs.parent`?

Reply:
fixed